content/developer/reference/user_interface/view_architectures.rst

```diff
@@ -2520,53 +2519,287 @@ Optional attributes can be added to the root element `kanban` to customize the v
+that can be toggled with a vertical ellipsis (:guilabel:`⋮`) on the top right of the card.
+
+The templates are written in :ref:`JavaScript QWeb <reference/qweb/javascript>`
 
 .. code-block:: xml
 
    <kanban>
-       <field name="FIELD_NAME"/>
-       ...
+      <templates>
+         <t t-name="card">
+            <field name="name"/>
+         </t>
+      </templates>
    </kanban>
 
-The `field` element can have the following attributes:
+Fields
+******
 
+Inside those templates, the `field` element allows to render a field. It can have the following
+attributes:
+ 
 .. include:: view_architectures/field_attribute_name.rst
 
-.. admonition:: Possible structure and representation of its rendering
+By default, field nodes are replaced by a `span` containing their formatted value, unless the
+`widget` attribute is specified, in which case their rendering and behavior depends on the
+corresponding widget. The `widget` attribute can have different values including:
 
-   .. list-table::
-      :class: o-showcase-table
+   .. attribute:: handle
+     :noindex:
 
-      * - .. image:: view_architectures/kanban_field.svg
-             :align: center
+     Allows reordering records with a drag and drop, using the corresponding field as order.
 
-      * - .. code-block:: xml
+   .. attribute:: kanban_color_picker
+     :noindex:
 
-             <kanban>
-                 <templates>
-                     <t t-name="kanban-box">
-                         <div>
-                             <field name="name"/>
-                         </div>
-                     </t>
-                 </templates>
-             </kanban>
+     Allows editing a color (integer) field. Combined with the root attribute `highlight_color`,
+     allows editing the color of the cards.
+
+See the :ref:`Field section <reference/js/widgets>` to discover
+various widgets and their options.
+
+Rendering Context
+*****************
+
+Kanban templates being rendered with the :ref:`QWeb engine <reference/qweb/javascript>`, they have
+a *rendering context* containing useful information and tools. Here're the available variables:
+
+.. attribute:: record
+   :noindex:
+
+   An object with all the fields defined in the view. Each field has two attributes: `value`
+   and `raw_value`. The former is formatted according to current user parameters while the latter is
+   the raw value (e.g. the `id` for a many2one field). This object is useful for instance to use
+   field values inside `t-if` conditions. For display purposes, we recommend using the `<field>` tag.
+
+   .. example::
+      .. code-block:: xml
+
+         <kanban>
+            <templates>
+               <field name="is_company"/>
+               <t t-name="card">
+                  <field name="name"/>
+                  <field t-if="!record.is_company.raw_value" name="parent_id">
+               </t>
+            </templates>
+         </kanban>
+
+.. attribute:: widget
```

Comment:
oups, this is where we see the weight of the past. would have been nice to rename that to something more explicitely linked to access rights. no big deal though